server/form-pages/apply/risk-and-need-factors/oasys-import/offenceDetails.ts

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-underscore-dangle */
+import type { DataServices, PersonRisksUI } from '@approved-premises/ui'
+
+import type { Application, ArrayOfOASysOffenceDetailsQuestions, OASysSections } from '@approved-premises/api'
+
+import TasklistPage from '../../../tasklistPage'
+
+import { Page } from '../../../utils/decorators'
+import { oasysImportReponse } from '../../../../utils/oasysImportUtils'
+
+type OffenceDetailsBody = {
+  offenceDetailsAnswers: Array<string> | Record<string, string>
+  offenceDetailsSummaries: ArrayOfOASysOffenceDetailsQuestions
+}
+
+@Page({
+  name: 'offence-details',
+  bodyProperties: ['offenceDetailsAnswers', 'offenceDetailsSummaries'],
+})
+export default class OffenceDetails implements TasklistPage {
+  title = 'Edit risk information'
+
+  offenceDetailsSummaries: OffenceDetailsBody['offenceDetailsSummaries']
+
+  offenceDetailsAnswers: OffenceDetailsBody['offenceDetailsAnswers']
+
+  risks: PersonRisksUI
+
+  constructor(public body: Partial<OffenceDetailsBody>) {}
+
+  static async initialize(
```

Comment:
Is there any sharing of logic we can do to DRY these initializers up a bit?

Reply:
Yeah I will add this in a subsequent PR